### ISSUE TITLE
Google Maps accessibility - iFrame title missing

### ISSUE THREAD

**Issue by cydfer** [elementor/elementor#9955]
- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
Similar to #9374, the Google Maps iFrame does not have the option to include a title attribute. I added an attribute for an ARIA label, but apparently that's not enough to keep from getting the "missing title in iframe" flag in accessibility testers.

**Describe the solution you'd like**
Please either automatically add a title attribute, or allow us to enter one in a field.

**Describe alternatives you've considered**
I thought about using an HTML widget to recreate the iframe, but that would be reinventing the wheel (abandoning the nice map functionality Elementor provided). I realized it couldn't be fixed with CSS and didn't see a way to use the attributes section under Advanced to create one.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.0.6

Feel free to update

Thanks!
